.circleci/config.yml

```diff
@@ -0,0 +1,134 @@
+---
+version: 2
+
+jobs:
+  test:
+    docker:
+    - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/prometheus/prometheus
+    resource_class: large
+
+    steps:
+    - checkout
+    - run: make promu
+    - run: make check_license style unused staticcheck build
+    - run: rm -v prometheus
```

Comment:
just curious why do we need this?

Reply:
We don't need to, but it's somewhat a pattern to double check that we actually built a binary.  `rm` will error if there's no file.

Reply:
ah , got it thank!